ibaqpy/bin/compute_tpa.py

```
@@ -0,0 +1,203 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+from matplotlib.backends.backend_pdf import PdfPages
+from pyopenms import *
```

Ruff (0.8.0)

`from pyopenms import *` used; unable to detect undefined names

(F403)